<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormObject</class>
 <widget class="QWidget" name="FormObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HPGCToolbox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="HPGCToolbox.qrc">
    <normaloff>:/tool</normaloff>:/tool</iconset>
  </property>
  <widget class="Line" name="line1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>581</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>目标函数</string>
    </attribute>
    <widget class="QTableWidget" name="tableObject">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>目标名称</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="HPGCToolbox.qrc">
        <normaloff>:/label</normaloff>:/label</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>表达式</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="HPGCToolbox.qrc">
        <normaloff>:/function</normaloff>:/function</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>监视</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="HPGCToolbox.qrc">
        <normaloff>:/eye</normaloff>:/eye</iconset>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnAddObject">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>30</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/addblack</normaloff>:/addblack</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDeleteObject">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>60</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/deleteblack</normaloff>:/deleteblack</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpObject">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>90</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/upblack</normaloff>:/upblack</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDownObject">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>120</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/downblack</normaloff>:/downblack</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>263</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>多目标处理方式</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbMutiObjectsFunction">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenMutiObjectsFunction">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>260</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/openfolder</normaloff>:/openfolder</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>约束条件</string>
    </attribute>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>263</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>多约束处理方式</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableLimit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>约束名称</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="HPGCToolbox.qrc">
        <normaloff>:/label</normaloff>:/label</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>表达式</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="HPGCToolbox.qrc">
        <normaloff>:/function</normaloff>:/function</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>运算符</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="HPGCToolbox.qrc">
        <normaloff>:/operator</normaloff>:/operator</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>阈值</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="HPGCToolbox.qrc">
        <normaloff>:/value</normaloff>:/value</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>监视</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="HPGCToolbox.qrc">
        <normaloff>:/eye</normaloff>:/eye</iconset>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnAddLimit">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>30</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/addblack</normaloff>:/addblack</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpLimit">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>90</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/upblack</normaloff>:/upblack</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDeleteLimit">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>60</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/deleteblack</normaloff>:/deleteblack</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDownLimit">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>120</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/downblack</normaloff>:/downblack</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenMutiLimitsFunction">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>260</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="HPGCToolbox.qrc">
       <normaloff>:/openfolder</normaloff>:/openfolder</iconset>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbMutiLimitsFunction">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnEncodeDataSource">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>14</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="HPGCToolbox.qrc">
     <normaloff>:/barcode</normaloff>:/barcode</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveConfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存配置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>17</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据源</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDataSource">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>16</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupIndicator">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>581</width>
     <height>266</height>
    </rect>
   </property>
   <property name="title">
    <string>指标元算子</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupParameter">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>15</y>
      <width>321</width>
      <height>215</height>
     </rect>
    </property>
    <property name="title">
     <string>参数设置</string>
    </property>
    <widget class="QWidget" name="widgetParameter" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>15</y>
       <width>310</width>
       <height>195</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeIndicator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>230</width>
      <height>210</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>indicator</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnAddIndicatorset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>235</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加指标集</string>
    </property>
    <property name="icon">
     <iconset resource="HPGCToolbox.qrc">
      <normaloff>:/indicatorset</normaloff>:/indicatorset</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddIndicator">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>235</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="HPGCToolbox.qrc">
      <normaloff>:/addblue</normaloff>:/addblue</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteIndicator">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>235</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="HPGCToolbox.qrc">
      <normaloff>:/deleteblue</normaloff>:/deleteblue</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOpenDataSource">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>14</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="HPGCToolbox.qrc">
     <normaloff>:/openfolder</normaloff>:/openfolder</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlgorithmParamSet">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>670</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>算法参数设置&gt;&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="HPGCToolbox.qrc"/>
 </resources>
 <connections/>
</ui>
